metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/data_classes.py

```diff
@@ -0,0 +1,200 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any, Dict, List, Optional
+
+from datahub.emitter.mcp_builder import PlatformKey
+
+
+class WorkspaceKey(PlatformKey):
+    workspace: str
+
+
+@dataclass
+class Workspace:
+    """
+    PowerBi Workspace
+    """
+
+    id: str
+    name: str
+    dashboards: List["Dashboard"]
+    reports: List["Report"]
+    datasets: Dict[str, "PowerBIDataset"]
```

Comment:
Minor consistency thing -- Why do we say Dashboard, Report, and then PowerBIDataset? 

Why not PowerBIDashboard?

Reply:
the concept of Datasets in PowerBI and DataHub is not the same. In PowerBI dataset is a container, so just to emphasize that difference I explicitly added PowerBI in the name of Dataset